src/main/java/com/hotels/housekeeping/service/impl/FileSystemHousekeepingService.java

```diff
@@ -47,39 +48,65 @@ public FileSystemHousekeepingService(LegacyReplicaPathRepository legacyReplicaPa
     LOG.warn("{}.fixIncompleteRecord(LegacyReplicaPath) should be removed in future.", getClass());
   }
 
+  private FileSystem fileSystemForPath(LegacyReplicaPath cleanUpPath) {
+    LOG.info("Attempting to delete path '{}' from file system", cleanUpPath);
```

Comment:
This isn't doing a delete? Update comment to clarify that this is trying to determine the FS for a path, or just remove it. If you do decide to keep it it's probably worth putting on debug rather than info.

Reply:
Moved it to more relevant place, aftermath of copy and paste :)